spikeinterface/sorters/launcher.py

```diff
@@ -225,14 +227,30 @@ def run_sorters(sorter_list,
     
     dtype_rec_name = np.dtype(type(list(recording_dict.keys())[0]))
     assert dtype_rec_name.kind in ("i", "u", "S", "U"), "Dict keys can only be integers or strings!"
-    
+
+    if sorter_params is not None:
```

Comment:
this allows to use the `run_sorters` to run the same sorter with different parameters :)